api-lib/images.ts

```diff
@@ -0,0 +1,48 @@
+import sharp from 'sharp';
+
+import { MAX_IMAGE_BYTES_LENGTH } from '../src/lib/images';
+
+import { PayloadTooLargeError } from './HttpError';
+
+export type ImageResizer = (imageBuffer: Buffer) => Promise<Buffer>;
+
+export function parseBase64Image(imageDataBase64: string): Buffer {
```

Comment:
should name this file `imageHelpers.ts` instead since its not actually images but a set of helpers to manipulate images